include/seastar/net/socket_defs.hh

```diff
@@ -123,6 +133,14 @@ struct ipv6_addr {
     }
 };
 
+struct ud_addr {
+    std::filesystem::path sfile_;
```

Comment:
i'd suggest s/sfile_/path/. and just use `std::string`.

Reply:
fixed